packages/manager/src/queries/cloudpulse/queries.ts

```diff
@@ -69,6 +70,10 @@ export const queryFactory = createQueryKeys(key, {
     }
   },
 
+  serviceTypes: (queryKey: string, serviceTypeKey: string) => ({
+    queryFn: () => getCloudPulseServiceTypes(),
+    queryKey: [queryKey, serviceTypeKey],
+  }),
```

Comment:
```suggestion
  serviceTypes : {
  queryFn: getCloudPulseServiceTypes,
  queryKey: null
  },
```

If api data is not dependent on the parameters passed then we can remove the parameters & replace with suggested logic